R/verb-compute.R

```diff
@@ -94,7 +94,7 @@ db_compute.DBIConnection <- function(con,
 #' @param warn_incomplete Warn if `n` is less than the number of result rows?
 #' @export
 collect.tbl_sql <- function(x, ..., n = Inf, warn_incomplete = TRUE) {
-  assert_that(length(n) == 1, n > 0L)
+  assert_that(length(n) == 1, n >= 0L)
```

Comment:
Maybe convert the if statement to `identical(n, Inf)` then `head()` can take care of the validation?